colour/colorimetry/spectrum.py

```diff
@@ -3690,9 +3939,10 @@ def interpolate(self, shape=SpectralShape(), method=None):
 
     def align(self,
               shape,
-              method='Constant',
-              left=None,
-              right=None):
+              interpolation_method=None,
+              extrapolation_method='Constant',
+              extrapolation_left=None,
```

Comment:
It would also create a discrepancy with the `interpolation_method` argument just above that accepts a string and also the way we use the `*_method` arguments in general across the API.

This is maybe an opportunity to review that as a whole?